tests/integration-tests/tests/efa/test_efa.py

```diff
@@ -54,7 +54,7 @@ def test_efa(
         multithreading_disabled = True
     else:
         head_node_instance = "c6g.16xlarge"
-        multithreading_disabled = False
+        multithreading_disabled = True
```

Comment:
what about removing this variable if there is only one case? Than you can also change the config to remove the placeholder

Reply:
Fixed in the next rev.